frontend/src/components/collective/CollectiveExpenseItem.js

```diff
@@ -39,6 +39,13 @@ const CollectiveExpenseItem = ({
 
   let src, srcSet, href, receiptMessage, receiptHelpTip;
 
+  let submittedByName = user && user.name;
+
+  // show email only if the user can view receipt
+  if (canViewReceipt) {
```

Comment:
I don't think everyone should be able to see the emails. I think everyone should see the name, yes. I personally wouldn't want to leak my email everywhere on the site.

`canViewReceipt` is different than `canEdit`. `canViewReceipt` includes the submitter of the receipt, in addition to the users that can `canEdit`.